internal/store/store.go

```diff
@@ -283,13 +285,16 @@ func (s *Store) setAtlasClient(client *http.Client) error {
 			respHeaders += fmt.Sprintf("%v: %v\n", key, strings.Join(value, " "))
 		}
 
+		respBody := &bytes.Buffer{}
+		_ = json.Indent(respBody, resp.Raw, "", "  ")
```

Comment:
[q] what if the response is not json? I think we have some plain text requests.

Reply:
I assumed the response would be always json 🤔  I can update the code to keep using `resp.Raw` if `json.Indent` return an error : )

Reply:
check the headers for content type

![disappear](https://media.giphy.com/media/a93jwI0wkWTQs/giphy.gif)